suites/pacific/cephadm/test-container-cli-args.yaml

```diff
@@ -0,0 +1,363 @@
+      desc: Add RGW services using spec file
+      module: test_cephadm.py
+      polarion-id: CEPH-83574640
+      config:
+        steps:                # create realm, zone group and zone
+          - config:
+              command: shell
+              args:
+                - "radosgw-admin realm create --rgw-realm=east --default"
+          - config:
+              command: shell
+              args:
+                - "radosgw-admin zonegroup create --rgw-zonegroup=asia --master --default"
+          - config:
+              command: shell
+              args:
+                - "radosgw-admin zone create --rgw-zonegroup=asia --rgw-zone=india --master --default"
+          - config:
+              command: shell
+              args:
+                - "radosgw-admin period update --rgw-realm=east --commit"
+          - config:
+              command: apply_spec
+              service: orch
+              specs:
+                - service_type: rgw
+                  service_id: my-rgw
+                  placement:
+                    count_per_host: 2
+                    nodes:
+                      - node4
+                      - node3
+                  spec:
+                    rgw_frontend_port: 8080
+                    rgw_realm: east
+                    rgw_zone: india
+                  extra_container_args:
+                    - "--cpus=1"
+          - config:
+              command: shell
+              args:              # sleep to get all services deployed
+                - sleep
+                - "120"
+  - test:
+      name: Configure client
+      desc: Configure client on node4
+      module: test_client.py
+      polarion-id:
+      config:
+        command: add
+        id: client.1                      # client Id (<type>.<Id>)
+        node: node4                       # client node
+        install_packages:
+          - ceph-common                   # install ceph common packages
+        copy_admin_keyring: true          # Copy admin keyring to node
+        store-keyring: true               # /etc/ceph/ceph.client.1.keyring
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+      destroy-cluster: false
+      abort-on-fail: true
+  - test:
+      name: Cephadm-ansible clients playbook
+      desc: Configure client node using cephadm-clients.yml playbook
+      polarion-id: CEPH-83574415
+      module: test_cephadm_ansible.py
+      config:
+        playbook: cephadm-clients.yml
+        extra-vars:
+          keyring: /etc/ceph/ceph.client.1.keyring
+          client_group: clients
+          fsid: f64f341c-655d-11eb-8778-fa163e914bcc
+  - test:
+      name: check-ceph-health on client node4
+      desc: Check for ceph health debug info
+      module: exec.py
+      config:
+        sudo: True
+        check_status: True
+        commands:
+          - "ceph status"
```

Comment:
Can we introduce some IO tests for client which is been created in previous test case.

Reply:
Done.